src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx

```diff
@@ -82,25 +83,42 @@ export class VanguardArtistWrapper extends React.Component<
     }
   }
 
-  getSVGBackground(index, section) {
+  getSVGBackground(index, section, isTest) {
     switch (section) {
```

Comment:
We could also check for `isTest` before getting into the switch, and use a new value over index-
`const i = isTest ? 1 : index`

Reply:
We can also grab `isTest` from props to avoid having to pass it into the function.